nvim_hunspell/builtin.lua

```diff
@@ -0,0 +1,39 @@
+vim.o.spelloptions = "hunspell"
+vim.o.spelllang = "en"
+vim.o.spell = true
+
+local total_time = os.time()
+local ITERATIONS = 1
+
+local errors = 0
+for i = 1, ITERATIONS do
+  local file = io.open("words", "r")
+  if not file then
+    error "Could not open wordlist"
+  end
+
+  errors = 0
+  for line in function() return file:read "*l" end do
+    local bad, suggs_plain = unpack(vim.split(line, "\t", { plain = true }))
+    print(bad)
+
+    local suggs = vim.split(suggs_plain, ', ', { plain = true, trimempty = true })
+
+    local bad_spell = vim.fn.spellbadword(bad)[1]
```

Comment:
Don't forget we also have `vim.spell.check` which avoids the viml bridge.

Reply:
Oh I forgot about that, I'll try it this way then.